src/main/scala/akka/io/AsyncDnsResolver.scala

```diff
@@ -123,8 +145,13 @@ class AsyncDnsResolver(cache: SimpleDnsCache, config: Config) extends Actor with
 object AsyncDnsResolver {
   private val inetSocketAddress = """(.*?)(?::(\d+))?""".r
 
+  case class SrvResolved(name: String, srv: immutable.Seq[SRVRecord])
```

Comment:
I'm being consistent with Akka here. For example `Dns.Resolve`. The records have departed that convention and have adopted DNS conventions, so I see the capitalisation as correct.